src/components/MediatedRequestsActivities/components/MediatedRequestsFilters/components/MediatedRequestStatusFilter/MediatedRequestStatusFilter.js

```diff
@@ -29,19 +51,34 @@ const MediatedRequestStatusFilter = () => {
         data-testid={MEDIATED_REQUEST_FILTER_TYPES.MEDIATED_REQUEST_STATUS}
         header={FilterAccordionHeader}
         label={formatMessage({ id:'ui-requests-mediated.filters.mediatedRequestStatus.accordionTitle' })}
-        name={MEDIATED_REQUEST_FILTER_TYPES.MEDIATED_REQUEST_STATUS}
+        name={filterPath}
         separator={false}
-        onClearFilter={() => {}}
+        displayClearButton={!isEmpty(activeValues)}
+        onClearFilter={() => clearGroup(filterPath)}
```

Comment:
Could you create separate function out of `return` statement?
Anonymous functions can make unnecessary rerenders.

Reply:
Fixed